django_elastic_migrations/migrations/0007_auto_20181111_1251.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-11-11 12:51
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('django_elastic_migrations', '0006_auto_20180709_2101'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='index',
+            name='active_version',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to='django_elastic_migrations.IndexVersion'),
```

Comment:
Django 2.0 requires explicit `on_delete` handlers for foreign keys. I've used the default, `models.CASCADE`, in all places except one: `Index.active_version`, which should just be set to null if the `IndexVersion` is deleted. This migration captures that change.